in_toto/runlib.py

```diff
@@ -571,7 +571,9 @@ def in_toto_run(name, material_list, product_list, link_cmd_args,
     signing_keyid = signature["keyid"]
     filename = FILENAME_FORMAT.format(step_name=name, keyid=signing_keyid)
     LOG.info("Storing link metadata to '{}'...".format(filename))
-    link_metadata.dump(filename)
+    if metadata_directory == None:
```

Comment:
In Python, we use `is` for this comparison (also caught by the linter FYI). `if metadata_directory is None:` basically. However, the `./` doesn't actually provide us with anything, so I recommend making this condition `if metadata_directory is not None:` and performing the directory check and join within to `filename`. If it indeed is `None`, we don't need to bother, and things will continue as they do now.